test/indentation-test-RRR.R

```diff
@@ -0,0 +1,349 @@
+
```

Comment:
I have no definite opinion on that. But given that etc is so cluttered and that these files are generated by a a script in test/ folder I think it would be better to leave them in test/ folder (for now at least).